packages/mui-material/package.json

```diff
@@ -89,6 +89,7 @@
   "peerDependencies": {
     "@emotion/react": "^11.5.0",
     "@emotion/styled": "^11.3.0",
+    "@mui/zero-runtime": "workspace:^",
```

Comment:
Not sure if this is the right approach but I think the current error only affects when used with pnpm.

Reply:
I think optional is needed as well?

```json
"peerDependenciesMeta": {
    …
    "@mui/zero-runtime": {
      "optional": true
    }
  },
```

Reply:
npm automatically install none optional peer dependencies https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/#user-content-exciting-new-features, leading to #40969. I think Jun is right.

Reply:
I did make the changes but seems I ended up pushing to a different branch. Fixing this.